docs/examples/plot_custom_parser.py

```diff
@@ -0,0 +1,286 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+.. _l-custom-parser:
+
+When a custom model has is nor a classifier neither a regressor
```

Comment:
When a custom model is neither a classifier nor a regressor.

Reply:
fixed